pkg/diagnostics/analyzers_test.go

```diff
@@ -26,3 +27,15 @@ func getDeploymentStatusAnalyzer(analyzers []*diagnostics.Analyze, name string)
 
 	return nil
 }
+
+func TestEksaLogTextAnalyzers(t *testing.T) {
+	collectorFactory := diagnostics.NewDefaultCollectorFactory()
+	collectors := collectorFactory.ManagementClusterCollectors()
+	analyzerFactory := diagnostics.NewAnalyzerFactory()
+	expectAnalzyers := analyzerFactory.EksaLogTextAnalyzers(collectors)
+	for _, analyzer := range expectAnalzyers {
+		if analyzer == nil {
+			t.Errorf("EksaLogTextAnalyzers: %s failed", analyzer.TextAnalyze.CheckName)
```

Comment:
What are we testing here? Just that the analyzer is not nil? In that case I would make that info part of the error message

Reply:
Eksa Log Text Analyzers will check namespaced log collectors are present and then return the analyzers associated with that namespace. So I would say it is testing if there is any returned analyzers. If error message is changed to "None analyzer returned, analyzer.TextAnalyze.CheckName.", do you think it make more sense?

Reply:
It won't catch specific bug but test the functionality. I have change the error message by removing out the `analyzer.TextAnalyzer.CheckName`.